doc/whats_new/v0.24.rst

```diff
@@ -368,6 +368,10 @@ Changelog
   `predict` and `score` would fail when `fit_intercept=False` and there was
   one feature during fitting. :pr:`18121` by `Thomas Fan`_.
 
+- |Fix| Fixes bug in :class:`linear_model.Perceptron` where `__init__` method
+  doesn't have `l1_ratio` parameter. :pr:`18622` by 
```

Comment:
I would suggest:

```suggestion
- |Fix| Added the missing `l1_ratio` parameter in :class:`linear_model.Perceptron`, to be used when `penalty='elasticnet'`. This changes the default from 0 to 0.15. :pr:`18622` by 
```

(make sure to properly break lines)

Reply:
Thanks for helping me on this PR.
I updated the fix entry. :)